Sources/Graphiti/Types/Field.swift

```diff
@@ -41,11 +41,15 @@ public class FieldBuilder<Root, Context, Type> {
 
     func addAllFields() throws {
         for property in try properties(Type.self) {
-            let field = GraphQLField(
-                type: try schema.getOutputType(from: property.type, field: property.key)
-            )
+            do {
```

Comment:
I think this might be more idiomatic like this:

```swift
for property in try properties(Type.self) {
    if let type = try? schema.getOutputType(from: property.type, field: property.key) {
        fields[property.key] = GraphQLField(type: type)
    }
}
```

On a related issue. I was thinking about not adding all field implicitly. Instead have a function like this:

```swift
try schema.object(type: Planet.self) { planet in
    try planet.exportFields(excluding: "mass", "population")
}
```

This way the reader knows exactly what's happening and you can also exclude fields you don't want to expose. What do you think?